docs/content/guides/developer/app-examples/trustless-swap.mdx

```diff
@@ -962,13 +956,90 @@ app.get('/escrows', async (req, res) => {
 });
 ```
 
+### Deployment {#deployment}
+
+Now that you have an indexer and an API service, you can deploy your move package and start the indexer and API service.
+
+1. Install dependencies by running
+
+```
+pnpm install --ignore-workspace
+```
+
+2. Setup the database by running
+
+```
+pnpm db:setup:dev
+```
+
+3. [Publish the contract & demo data](#demo-data)
+
+4. Run both the API and the indexer
+
+```
+pnpm dev
+```
+
+5. Visit [http://localhost:3000/escrows](http://localhost:3000/escrows) or [http://localhost:3000/locked](http://localhost:3000/locked)
+
+### Demo Data<a name="demo-data"></a> {#demo-data}
+
+{@include: ../../../snippets/initialize-sui-client-cli.mdx}
+
+Next, configure the Sui CLI to use `testnet` as the active environment, as well. If you haven't already set up a `testnet` environment, do so by running the following command in a terminal or console:
+
+```bash
+sui client new-env --alias testnet --rpc https://fullnode.testnet.sui.io:443
+```
+
+Run the following command to activate the `testnet` environment:
+
+```bash
+sui client switch --env testnet
+```
+
+Before being able to publish your package to Testnet, you need Testnet SUI tokens. To get some, join the [Sui Discord](https://discord.gg/Sui), complete the verification steps, enter the `#testnet-faucet` channel and type `!faucet <WALLET ADDRESS>`. For other ways to get SUI in your Testnet account, see [Get SUI Tokens](/guides/developer/getting-started/get-coins).
+
+Now that you have an account with some Testnet SUI, you can deploy your contracts. 
+
+There are some helper functions to:
+
+1. Publish the smart contract
+2. Create some demo data (for testnet)
+
+To produce demo data:
+
+1. Publish the smart contract by running
+
+```
+npx ts-node helpers/publish-contracts.ts
+```
+
+2. Produce demo non-locked and locked objects
+
+```
+npx ts-node helpers/create-demo-data.ts
+```
+
+3. Produce demo escrows
+
+```
+npx ts-node helpers/create-demo-escrows.ts
+```
+
+If you want to reset the database (start from scratch), run:
```

Comment:
Useful to explaln why might you want to do this? And/or provide a way to check db?